include/seastar/core/sharded.hh

```diff
@@ -784,6 +759,7 @@ sharded<Service>::stop() noexcept {
 }
 
 template <typename Service>
+inline
```

Comment:
inlines aren't necessary for templates, and are a net negative for this compile time oriented function (though in practice, the compiler ignores the inline directive for inlining).

Reply:
right, fixing

Reply:
will remove inline from templates in sharded.hh on a different PR